src/main/java/hudson/plugins/ec2/ssh/EC2UnixLauncher.java

```diff
@@ -411,9 +416,9 @@ public boolean verifyServerHostKey(String hostname, int port, String serverHostK
         }
     }
 
-    private String getEC2HostAddress(EC2Computer computer) throws InterruptedException {
+    private static String getEC2HostAddress(EC2Computer computer, SlaveTemplate template) throws InterruptedException {
```

Comment:
Why static? I cannot see when it's needed

Reply:
I changed it to static because it doesn't need any value in the actual class itself so it should have been static from the start as its not required to be tied to any instance of the Launcher

Reply:
your words make sense, but it looks weird to me having a static private method which is not called within a static block of code. Anyway, I'm not blocking the PR because of this. It's just matter of taste and I only wanted to know if there were other reasons. I will let the others give their opinion if applies and I'm approving the PR.

Reply:
Ahh for me I used to mostly write C++, so a static method has the benefit of not passing in an additional parameter(The this pointer). So the habit stayed with me.